go/chat/unfurl/scrape_generic_scoring.go

```diff
@@ -169,6 +169,26 @@ var attrRankMap = map[string]attrRanker{
 		score:   getOpenGraphScore,
 		setter:  setPublishTime,
 	},
+	"pdate": attrRanker{
+		content: getContentAttr,
+		score:   getDefaultScore,
+		setter:  setPublishTime,
+	},
+	"article.published": attrRanker{
+		content: getContentAttr,
+		score:   getOpenGraphScore,
+		setter:  setPublishTime,
+	},
+	"article:published": attrRanker{
+		content: getContentAttr,
+		score:   getOpenGraphScore,
```

Comment:
`getDefaultScore`

Reply:
I think these should be equal weight as OG (or at least higher than default). Would you prefer having something in between OG and default, or just have these the same?